src/components/contextual/pages/pool/invest/InvestPage.vue

```diff
@@ -15,21 +15,23 @@ const { upToLargeBreakpoint } = useBreakpoints();
 </script>
 
 <template>
-  <div class="invest-page-layout-grid">
-    <div v-if="!upToLargeBreakpoint" class="col-span-5">
-      <InvestPageMyWallet />
-    </div>
+  <Transition appear name="appear">
```

Comment:
On pages like this, the only thing that changed was adding the <Transition></Transition> tags.